instrumentation/opentelemetry-instrumentation-django/tests/test_middleware.py

```diff
@@ -517,6 +715,105 @@ def test_wsgi_metrics(self):
+                                duration_s, point.sum, places=2
+                            )
+                        if isinstance(point, NumberDataPoint):
+                            number_data_point_seen = True
+                            self.assertEqual(point.value, 0)
+                        for attr in point.attributes:
+                            self.assertIn(
+                                attr, _recommended_attrs[metric.name]
+                            )
+        self.assertTrue(histrogram_data_point_seen and number_data_point_seen)
+
+    # pylint: disable=too-many-locals
+    # pylint: disable=too-many-nested-blocks
+    def test_wsgi_metrics_both_semconv(self):
+        _expected_metric_names = [
+            "http.server.duration",
+            "http.server.active_requests",
+            "http.server.request.duration",
+        ]
+        active_count_both_attrs = list(_server_active_requests_count_attrs_new)
+        active_count_both_attrs.extend(_server_active_requests_count_attrs_old)
+        _recommended_attrs = {
+            "http.server.active_requests": active_count_both_attrs,
+            "http.server.request.duration": _server_duration_attrs_new,
+            "http.server.duration": _server_duration_attrs_old,
+        }
+        start = default_timer()
+        for _ in range(3):
+            response = Client().get("/span_name/1234/")
+            self.assertEqual(response.status_code, 200)
+        duration = max(round((default_timer() - start) * 1000), 0)
+        duration_s = max(default_timer() - start, 0)
+        metrics_list = self.memory_metrics_reader.get_metrics_data()
+        number_data_point_seen = False
+        histrogram_data_point_seen = False
+
+        self.assertTrue(len(metrics_list.resource_metrics) != 0)
+        for resource_metric in metrics_list.resource_metrics:
+            self.assertTrue(len(resource_metric.scope_metrics) != 0)
+            for scope_metric in resource_metric.scope_metrics:
+                self.assertTrue(len(scope_metric.metrics) != 0)
+                for metric in scope_metric.metrics:
+                    self.assertIn(metric.name, _expected_metric_names)
+                    data_points = list(metric.data.data_points)
+                    self.assertEqual(len(data_points), 1)
+                    for point in data_points:
+                        if isinstance(point, HistogramDataPoint):
+                            self.assertEqual(point.count, 3)
+                            histrogram_data_point_seen = True
+                            if metric.name == "http.server.request.duration":
+                                self.assertAlmostEqual(
+                                    duration_s, point.sum, places=1
+                                )
+                            elif metric.name == "http.server.duration":
+                                self.assertAlmostEqual(
+                                    duration, point.sum, delta=10
```

Comment:
Any reason why deltas are different from the `new_semconv` and `default_semconv` tests?